tel2puml/find_unique_graphs/config.yaml

```diff
@@ -0,0 +1,15 @@
+data_sources:
+  json:
+    type: JSONDataSource
+    path: /path/to/your/json/directory/or/file
+
+data_holders:
+  sql:
+    type: SQLDataHolder
```

Comment:
I'm wondering if we need this as the sql key should be enough info to specify this I think?